test/suite/unit/AttributeTable.spec.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1); // @codeCoverageIgnore
+
+namespace Rinq;
+
+describe(AttributeTable::class, function () {
```

Comment:
I think I mentioned in a previous PR to just axe `AttributeTable` completely, just use an associative array for great justice.